### ISSUE TITLE
no panel indicator in gnome flashback(metacity) 3.18

### ISSUE THREAD

**Issue by nilands55s** [Keruspe/GPaste#152]
This new Ubuntu 16.04 seems to work in the default Gnome login but not the one I like Gnome flashback Metacity. The other Gnome desktops do not have things I like in the top and bottom panels. I use the workspace switcher (in a row of 10-14 at the bottom-right) with is drag -n- drop, constantly and I have more control of the layout. I use gpaste a lot and would like my easy to use indicator back.
I have done some hacking at it but I am not a programmer and the last time I tried to compile for the desktop I FUBAR'd my desktop and had to start over reinstalling all.
![screenshot from 2016-05-05 22-47-24](https://cloud.githubusercontent.com/assets/8949704/15062925/6e38244a-1313-11e6-98b0-27b33f07be89.png)
This is on a new to me box HP xw9440 Workstation with Nvidia Quadro FX 3700/PCIe/SSE2 and Quad-Core AMD Opteron(tm) Processor 2380 × 8 with 16Gb RAM
The Ubuntu Server 16.04 had gnome desktop added to it as I don't want to do everything in the CLI.
Thanks James Niland
nilandenterprise.com

**Comment by Keruspe** [comment]
You will need gpaste-applet for that. Don't know if that's packaged for ubuntu but it definitely exists and is part of GPaste.

Note though that now, this kind of indicators are deprecated, so the only thing it does it launch gpaste-ui (which you can launch by hitting <Ctrl><Alt><G> too (that's the default which can e changed)

**Comment by nilands55s** [comment]
Well I have tried endlessly to get the Panel Icon to load in ubuntu-gnome destop 16.04.6 and there is no function to be had for the gnome 3.18. From what I have learned gnome 3.18 seems to have been abandon or never finished. No indicater in gnome shell or with the gpaste-applet, indicator, but will load in unity. I gave up troublshooting 2 decades ago because it litterly make me hostile. But if I did I have no clue how it is called or where to start as its workings
